spec/services/courses/copy_to_provider_service_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   it "makes a copy of the course in the new provider" do
-    service.execute(course: course, new_provider: new_provider)
+    service.execute(course: course, new_provider: new_provider, force: false)
```

Comment:
Why did we remove the default value and then have to start passing it everywhere?

Reply:
It broke this for me 
The principle of least astonishment 

passing a `force = true` from the outer layer in i would of expected it to populate down to inner layer  

As a `false is a false` as well as `silent is false` so why  `silent`?

https://github.com/DFE-Digital/teacher-training-api/blob/master/app/services/providers/copy_to_recruitment_cycle_service.rb#L33

https://github.com/DFE-Digital/teacher-training-api/blob/master/spec/services/courses/copy_to_provider_service_spec.rb#L190-L207 

Enforce by the fact that there is test for it but no path of execution

Reply:
Not to force is just the default operation so there's no need to pass a value. I agree you'd expect it to pass through to the inner service although as I've said, having the provider service call the course one is probably why all of this is getting so complicated.